setup.py

```diff
@@ -63,6 +63,12 @@ def main():
     log_rotate_dir = '/etc/logrotate.d'
     log_rotate_files = ['scripts/apel-client']
 
+    # Python 2.x dependencies
+    _install_requires = ['MySQL-python', 'iso8601', 'python-ldap', 'dirq', 'future']
```

Comment:
Can you flip this so that Python 3 is the default set and Python 2 modifies it? Then it'll be easier to remove in future.

Reply:
Sure. Make sense added in 45852290173e7bd4e79ea067e58efe3de4454d6c